dataplane/linux/endpoint_mgr_test.go

```diff
@@ -135,11 +135,19 @@ func chainsForIfaces(ifaceMetadata []string,
 	epMarkMapper rules.EndpointMarkMapper,
 	host bool,
 	tableKind string) []*iptables.Chain {
+	const (
+		ProtoUDP  = 17
+		ProtoIPIP = 4
+		VXLANPort = 0
```

Comment:
Why isn't this 4789 (which I think is the Felix default)?

Reply:
Oh, I guess it's because you don't initialize that field in the Config struct.  I think it would be more realistic to do that initialization; or else to add a comment here to explain why it's 0.

Reply:
Fixed

Reply:
Fixed